kafka-connect/src/main/java/io/tabular/iceberg/connect/data/Utilities.java

```diff
@@ -52,15 +55,31 @@ public class Utilities {
 
   public static Catalog loadCatalog(IcebergSinkConfig config) {
     return CatalogUtil.buildIcebergCatalog(
-        config.getCatalogName(), config.getCatalogProps(), getHadoopConfig());
+        config.getCatalogName(),
+        config.getCatalogProps(),
+        getHadoopConfig(config.getHadoopProps()));
   }
 
-  private static Object getHadoopConfig() {
-    try {
-      Class<?> clazz = Class.forName("org.apache.hadoop.conf.Configuration");
-      return clazz.getDeclaredConstructor().newInstance();
-    } catch (ClassNotFoundException e) {
+  private static Object getHadoopConfig(Map<String, String> hadoopProps) {
```

Comment:
I'm probably just missing something but how do the `hadoopProps` actually get used here (I don't see any references to it in the method)?

Reply:
Ah nvm, ignore I see it on line 80